run_gke.sh

```diff
@@ -53,4 +53,6 @@ kubectl run performance-tests -it --command --rm --quiet --generator=run-pod/v1
     --env=RABBITMQ_PASSWORD=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-password}" | base64 --decode) \
     --env=RABBITMQ_MAN_PORT=15672 \
     --env=SAMPLE_FILE_PATH=resources/sample_files/1_per_treatment_code.csv \
+    --env=SAMPLE_BUCKET=census-rm-performance-test-files \
```

Comment:
This doesn't match the name of the bucket in the terraform PR. Might need to tweak this?